Fabric.Identity.API/Persistence/SqlServer/Stores/SqlServerUserStore.cs

```diff
@@ -90,19 +81,21 @@ public void UpdateUser(User user)
 
             user.ToEntity(existingUser);
```

Comment:
Is there a case where claims can be removed?

Reply:
Good point, i've updated the code to handle this situation, and also added tests to cover it.